src/java/picard/illumina/IlluminaBasecallsToFastq.java

```diff
@@ -487,7 +482,7 @@ public void encode(final FastqRecordsForCluster val) {
             if (numSampleBarcodes != val.sampleBarcodeRecords.length) throw new IllegalStateException();
             encodeArray(val.templateRecords);
             encodeArray(val.sampleBarcodeRecords);
- //           encodeArray(val.molecularBarcodeRecords);
+            encodeArray(val.molecularBarcodeRecords);
```

Comment:
Could you please confirm that you did mean to uncomment this?  Because IlluminaBasecallsToFastq is a core tool, I want to make sure this was on purpose.